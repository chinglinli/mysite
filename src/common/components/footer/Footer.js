import React from 'react'

import styles from './Footer.scss'

export default class Footer extends React.Component {
  render () {
    return (
      <div className={styles.root}>
        <div className={styles.copyright}>Copyright © 2017 - 兩大類</div>
      </div>
    )
  }
}
