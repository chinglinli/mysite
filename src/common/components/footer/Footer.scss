@import '../../css/colors.scss';

.root {
  height: 2.8125rem;
  background: $dominant-color2;
  display: flex;
  justify-content: center;
  align-items: center;

  .copyright {
    color: white;
  }
}
