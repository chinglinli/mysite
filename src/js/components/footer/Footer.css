@value dominant-color2 from '../../css/colors.css';


.root {
    height: 45px;
    background: dominant-color2;
}
