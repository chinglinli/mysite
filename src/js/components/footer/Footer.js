import * as React from 'react';
import styles from './Footer.scss';

export default class Footer extends React.Component {
  render() {
    return (
      <div className={styles.root}>
        <div className={styles.copyright}>Copyright &copy; 2016 - 兩大類</div>
      </div>
    );
  }
}
