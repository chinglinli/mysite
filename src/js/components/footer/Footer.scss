@import '../../../css/colors.scss';


.root {
    height: 45px;
    background: $dominant-color2;

    .copyright {
        color: white;
        height: 100%;
        padding-top: 0.5em;
        text-align: center;
    }
}
