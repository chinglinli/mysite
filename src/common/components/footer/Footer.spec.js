import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';


test('It should render Footer component correctly', () => {
  const component = renderer.create(
    <Footer />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
});