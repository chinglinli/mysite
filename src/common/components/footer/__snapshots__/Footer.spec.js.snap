// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render Footer component correctly 1`] = `
<div
  className="root"
>
  <div
    className="copyright"
  >
    Copyright © 2017 - 兩大類
  </div>
</div>
`;
